ja_JP/LC_MESSAGES/login.po

```diff
@@ -242,5 +242,5 @@ msgid "SEARCH_FOR_SENDER_EMAIL_FORMAT"
 msgstr "メールを受信していないませんか？迷惑メールをチェックし、%sを探してみてください。 もし迷惑メールとしてマークされていたら、 メールの受信許可の設定をしてください。"
 
 msgid "NEW_USER_INTRO_POST_INTRO_CHANNEL"
-msgstr "When you're ready, go ahead and introduce yourself in our Introduction channel, under the Personal category."
+msgstr "準備が整ったら、先へ進み、個人カテゴリの下の紹介チャンネルで自己紹介をしていってください。"
```

Comment:
The channel name as displayed on the DSi is "Introductions" no matter the selected language.